buy-bitcoin-redirect.html

```diff
@@ -38,11 +38,7 @@
   <div class="lds-dual-ring"></div>
   <script>
     var safello_countries = ["SE"];
-    var xanpool_countries = ["TH", "PH", "IN", "VN", "MY"]; // + "SG", "HK", "ID"
-    var moonpay_currencies = ["AUD", "CAD", "CHF", "EUR", "GBP", "ILS", "NOK", "PLN", "RUB", "SEK", "USD", "ZAR"]; // curl https://api.moonpay.io/v3/currencies?apiKey=pk_live_IkhSI2lIXSiolwakfd95QFD4p3908cZa | jq '.' | grep fiat -A 2 | grep code
-    var xanpool_currencies = ["SGD", "HKD", "THB", "PHP", "INR", "IDR", "VND", "MYR", "SGD", "HKD", "PHP", "INR", "VND", "THB"];
-    var mercuryo_countries = ["RU", "UA", "BY", "AM", "AZ", "BY", "GEN", "KZ", "KE", "LA", "MD", "MC", "ME", "SM", "RS", "TJ", "UZ", "SG", "HK", "KR", "AR", "ID", "TR", "JP"];
```

Comment:
lets not remove partners irreversibly and leave an option to quickly enable them if needed.
the ez way to do this is make a list of their countries empty. 

`var mercuryo_countries = [];`

`var xanpool_countries = [];`